### ISSUE TITLE
Improve copy for architecture list

### ISSUE THREAD

**Issue by nedrichards** [flathub/frontend#177]
We should consider new copy for the architecture list on an app page. e.g. " 'aarch64, arm, i386, x86_64'" on a page like this https://flathub.vercel.app/apps/details/org.gnome.Connections

Something like aarch64 is quite arcane to even technically inclined users. I can't immediately think of better copy but will continue to think.

We should also think about how to highlight if an app is available on the architecture of the device you're browsing from, or perhaps more importantly not available in the context of aarch64 devices.

**Comment by bilelmoussaoui** [comment]
> We should also think about how to highlight if an app is available on the architecture of the device you're browsing from

That sounds like a great approach

And even if we keep the current architectures list as is, we should definitely blacklist the deprecated ones. eg i386 because most of those apps that are still built against that architecture and are definitely outdated, either directly or indirectly  (old unsupported runtime)

**Comment by razzeee** [comment]
I looked into highlighting back when we shipped this. Tldr is, won't work
or be very very unreliable. Should be in the issue tracker somewhere.
(could be in the old one)

Bilal Elmoussaoui ***@***.***> schrieb am Mo., 31. Jan. 2022,
22:18:

**Comment by razzeee** [comment]
https://github.com/flathub/frontend/issues/47

**Comment by nedrichards** [comment]
Thanks for that, might only be possible as a progressive enhancement then. We should have a separate ticket to denylist unsupported architectures as well - that's a good one @bilelmoussaoui

**Comment by razzeee** [comment]
I'm also a bit confused by the usage of the word `copy` in this context. Is `wording` what your after?

**Comment by nedrichards** [comment]
Yes, sorry for the 'englishism' it's items 3 and 5 in the noun definition here https://en.wiktionary.org/wiki/copy commonly used for 'the words that go into a UI, or a newspaper etc.'
